include/odp/api/spec/classification.h

```diff
@@ -751,7 +751,8 @@ odp_queue_t odp_cls_hash_result(odp_cos_t cos, odp_packet_t packet);
  * Discard a class-of-service along with all its associated resources
  *
  * Before destroying a CoS, all the PMRs referring to the CoS (as a source or
- * destination CoS) must be destroyed first.
+ * destination CoS) must be destroyed first. The CoS must not be in use as the
+ * default CoS in any pktio (@see odp_pktio_default_cos_set()).
```

Comment:
It could also be mentioned that the CoS must not be in use as the destination CoS of any IPsec SA.

Reply:
v2.